### ISSUE TITLE
Tab shrink: Close editor icon is invisible when Tab Key Moves Focus

### ISSUE THREAD

**Issue by usernamehw** [Microsoft/vscode#41459]
1. Open non-dirty file (New Untitled will do)
2. `Tab Key Moves Focus`
3. Tab to "close editor"
4. Focus border is set, which is nice, but it's invisible
Shown only on hover:
![close_tab](https://user-images.githubusercontent.com/9638156/34816194-3ceeee54-f6c5-11e7-808c-61c854047863.png)

**Comment by bpasero** [comment]
@usernamehw I cannot reproduce, can you reproduce with our default dark theme?

**Comment by usernamehw** [comment]
Yes.

**Comment by usernamehw** [comment]
I'm talking about the icon itself. Everything is logical, icon is invisible on saved files as usual.

But when you move focus by "tabbing" your border is disappearing going through tabs (because the icon is invisible).

**Comment by usernamehw** [comment]
I couldn't reproduce on linux VM, so maybe it's related to Windows

**Comment by usernamehw** [comment]
Ok, it's somehow related to config file. If I comment everything in file issue not reproduces.

**Comment by usernamehw** [comment]
I no longer able to reproduce

**Comment by usernamehw** [comment]
@bpasero, it's related to config. Reproduces with: `"workbench.editor.tabSizing": "shrink",`

**Comment by usernamehw** [comment]
I guess it's related with this piece:
```css
.monaco-workbench>.part.editor>.content>.one-editor-silo>.container>.title .tabs-container>.tab.close-button-right.sizing-shrink>.tab-close {
    flex: 0;
    overflow: hidden; /* <- here */
}
```

**Comment by bpasero** [comment]
@usernamehw thanks I can reproduce, but the fix is not good. I commented.

**Comment by usernamehw** [comment]
I think it would be easy to do in chrome 60 where `:focus-within` is working.
